src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (top, right, title, description, type) => {
+  const div = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  div.className = (`notification ${type}`);
+  div.style.cssText = `top: ${top}; right: ${right};`;
+  div.append(h2);
+  div.append(p);
+  h2.className = ('title');
+  h2.textContent = `${title}`;
```

Comment:
```suggestion
  h2.textContent = title;
```
title и так строка, не нужно приводить еще раз к строке (то же самое с description)